libs/ledger-live-common/src/__tests__/test-helpers/bridgeSetupUpdateMocks.ts

```diff
@@ -0,0 +1,53 @@
+import { PerformanceObserver, PerformanceObserverCallback } from "node:perf_hooks";
+import {
+  HttpPerformanceEntry,
+  HttpPerformanceEntryRequest,
+  HttpPerformanceEntryResponse,
+  StdRequest,
+} from "./types";
+import { createHash } from "crypto";
+
+function sha256(str: string) {
+  return createHash("sha256").update(str).digest("hex");
```

Comment:
I think base64 is enough. I we don't really need sha256 to encode it.

Reply:
Oh ok, will try :)